package.json

```diff
@@ -17,7 +17,7 @@
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.8.3",
```

Comment:
Please do not add useless files to your PR.
You can use **git add src/** command to add needed files
**git status** command — for checking file status